tests/data/drogon/ertrun1/share/metadata/fmu_case.yml

```diff
@@ -3,7 +3,7 @@ source: fmu
 version: 0.8.0
 class: case
 tracklog:
-  - datetime: "2021-05-05T09:20:40.475110"
+  - datetime: "2021-05-05T09:20:40.475110Z"
```

Comment:
Same thing for the case metadata in this test dataset.. we should look into auto updating it or create it on the fly in the tests instead.